src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.data.tag;
+
+import java.util.List;
+
+/**
+ * Class to associate a tag with a tag state and name.
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`